# Chapter 4 The Standard Deviation and Other Measures of Dispersion

## The Standard Deviation
The _standard deviation_ of a set of $N$ numbers $X_1,X_2,\ldots,X_N$ is denoted by $s$ and is defined by<p>
\begin{align}
s
= \sqrt{\frac{\sum^N_{j=1}{(X_j - \bar{X})^2}}{N}}
= \sqrt{\frac{\sum{(X=\bar{X})^2}}{N}}
= \sqrt{\frac{\sum{x^2}}{N}}
= \sqrt{(X - \bar{X})^2}\label{eq5}\tag{5}
\end{align}<p>
where $x$ represents the deviations of each of the numbers $X_j$ from the mean $\bar{X}$. Thus $s$ is the root mean square (RMS) of the deviations from the mean, or, as it is sometimes called, the _root-mean-square deviation_.<p>
If $X_1,X_2,\ldots,X_K$ occur with frequencies $f_1,f_2,\ldots,f_k$, respectively, the standard deviation can be written<p>
\begin{align}
s
=\sqrt{\frac{\sum^K_{j=1}{f_j(X_j-\bar{X})^2}}{N}}
=\sqrt{\frac{\sum{f(X-\bar{X})^2}}{N}}
=\sqrt{\frac{\sum{fX^2}}{N}}
=\sqrt{\overline{(X-\bar{X})^2}}\label{eq6}\tag{6}
\end{align}<p>
where $N=\sum^K_{j=1}{f_j}=\sum{f}$. In this form it is useful for grouped data.<p>
Sometimes the standard deviation of a sample's data is defined with $(N-1)$ replacing $N$ in the denominators of the expressions in equations (\ref{eq5}) and (\ref{eq6}) because the resulting value represents a better estimate of the standard deviation of a populiation from which the sample is taken. For large values of $N$ (certainly $N>30$), there is practically no difference between the two definitions. Also, when the better estimate is needed we can always obtain it by multiplying the standard deviation computed according to the first definition by $\sqrt{\frac{N}{(N-1)}}$. Hence we shall adhere to the form (5) and (6).

In [1]:
sd <- function(x) {
        return (sqrt(sum((x - mean(x))^2)/length(x)))
}

---

__4.9.__ Find the standard deviation $s$ of each set of numbers in Problem 1.

In [2]:
p1.a <- c(12, 6, 7, 3, 15, 10, 18, 5)
p1.b <- c(9, 3, 8, 8, 9, 8, 9, 18)
(p9.a.s <- round(sd(p1.a),2))
(p9.b.s <- round(sd(p1.b),2))

[1] 4.87

[1] 3.87

---

__4.10.__ The standard deviations of the two data sets given in Problem 4.1 were found using MINITAB and the results are shown below. Compare the answers with those obtained in Problem 4.9.<p>
```
MTB > print c1
set1
      12   6    7    3    15   10  18    5
MTB > print c2
set2
      9    3    8    8    9    8    9    18
MTB > standard deviation c1
```
Column Standard Deviation
```
    Standard deviation of set1 = 5.21
MTB > standard deviation c2
```
Column Standard Deviation
```
    Standard deviation of set2 = 4.14
```

### Solution
The MINITAB package uses the formula
$$
\begin{align*}
s = \sqrt{\frac{\sum{X-\bar{X})^2}}{N-1}}
\end{align*}
$$<p>
and therefore the standard deviations are not the same in Problem 4.9 and 4.10. The answers in Problem 4.10 are obtainable from those in problem 4.9 if we multiply those in Problem 4.9 by $\sqrt{\frac{N}{(N-1)}}$. Since $N=8$ for both sets $\sqrt{\frac{N}{(N-1)}}=1.069045$, and for set $1$, we have $(1.069045)(4.87)=5.21$, the standard deviation given by MINITAB. Similarly, $(1.069045)(3.87)=4.14$, the standard deviation given for set $2$ by MINITAB.

__When should the SD be computed with a denominator of n instead of (n-1)?__

Statistics books often show two equations to compute the SD, one using n, and the other using n-1, in the denominator. Some calculators have two buttons. 

The n-1 equation is used in the common situation where you are analyzing a sample of data and wish to make more general conclusions. The SD computed this way (with n-1 in the denominator) is your best guess for the value of the SD in the overall population.

If you simply want to quantify the variation in a particular set of data, and don't plan to extrapolate to make wider conclusions, then you can compute the SD using n in the denominator. The resulting SD is the SD of those particular values. It makes no sense to compute the SD this way if you want to estimate the SD of the population from which those points were drawn. It only makes sense to use n in the denominator when there is no sampling from a population, there is no desire to make general conclusions. 

The goal of science is always to generalize, so the equation with n in the denominator should not be used. The only example I can think of where it might make sense is in quantifying the variation among exam scores. But much better would be to show a scatterplot of every score, or a frequency distribution histogram. 

GraphPad Prism and InStat always compute the SD with the n-1 denominator.

---

__4.11.__ Find the standard deviation of the heights of the 100 male students at XYZ University (Table 2.1).

### Solution
From Problem 3.15, 3.20, or 3.22, $\bar{X}=67.45$in. The work can be arranged as in Table 4.2.<p>
\begin{align}
s
=\sqrt{\frac{\sum{f(X-\bar{X})^2}}{N}}
=\sqrt{\frac{852.7500}{100}}
=\sqrt{8.5275}=2.92\text{in}
\end{align}

In [3]:
load("Saves/Table_2.1.Rdata")
p11.f <- as.vector(Table_2.1)
p11.X <- c(61, 64, 67, 70,73)
p11.X.bar <- round(sum(p11.f*p11.X)/sum(p11.f),2)
p11.X.minus.X.bar <- round((p11.X - p11.X.bar),2)
p11.X.minus.X.bar.squared <- p11.X.minus.X.bar^2
p11.f.X.minus.X.bar.squared <- p11.f*p11.X.minus.X.bar.squared
p11.table <- matrix(c(p11.X, p11.X.minus.X.bar, p11.X.minus.X.bar.squared, p11.f, p11.f.X.minus.X.bar.squared), nrow=length(p11.f))
rownames(p11.table) <- rownames(Table_2.1)
colnames(p11.table) <- c("Class Mark (X)", "X - \\bar{X} = X - 67.45", "(X - \\bar{X})^2", "Frequency (f)", "f(X - \bar{X})^2")
p11.table

,Class Mark (X),X - \bar{X} = X - 67.45,(X - \bar{X})^2,Frequency (f),f(X - ar{X})^2
60-62,61,-6.45,41.6025,5,208.0125
63-65,64,-3.45,11.9025,18,214.2450
66-68,67,-0.45,0.2025,42,8.5050
69-71,70,2.55,6.5025,27,175.5675
72-74,73,5.55,30.8025,8,246.4200


In [4]:
(p11.sd <- round(sqrt(sum(p11.f*(p11.X-p11.X.bar)^2)/sum(p11.f)),2))

[1] 2.92

---

## The variance
The _variance_ of a set of data is defined as the square of the standard deviation and is thus given by $s^2$ in equations (5) and (6).<p>
When it is necessary to distinguish the standard deviation of a population from the standard deviation of a sample drawn from this population, we often use the symbol $s$ for the latter and $\sigma$ (lowercase Greek _sigma_) for the former. Thus $s^2$ and $\sigma^2$ would represent the _sample variance_ and _population variance_, respectively.

---

## Short methods for computing the standard deviation
Equations (5) and (6) can be written, respectively, in the equivalent forms
\begin{align}
s
=\sqrt{\frac{\sum^N_{j=1}{X_j^2}}{N}-\left(\frac{\sum^N_{j=1}{X_j}}{N}\right)^2}
=\sqrt{\frac{\sum{X^2}}{N}-\left(\frac{\sum{X}}{N}\right)^2}
=\sqrt{\overline{X^2}-\bar{X}^2}\label{eq7}\tag{7}
\end{align}<p>
\begin{align}
s
=\sqrt{\frac{\sum^K_{j=1}{f_jX_j^2}}{N}-\left(\frac{\sum^N_{j=1}{X_j}}{N}\right)^2}
=\sqrt{\frac{\sum{fX^2}}{N}-\left(\frac{\sum{fX}}{N}\right)^2}
=\sqrt{\overline{X^2}-\bar{X}^2}\label{eq8}\tag{8}
\end{align}<p>
where $\overline{X^2}$ denotes the mean of the squares of the various values of $X$, while $\bar{X}^2$ denotes the square of the mean of the various values of X (see Problems 4.12 to 4.14).<p>
If $d_j=X_j-A$ are the deviations of $X_j$ from some arbitrary constant A, results (\ref{eq7}) and (\ref{eq8}) become, resqpectively,<p>
\begin{align} 
s
=\sqrt{\frac{\sum^N_{j=1}{d_j^2}}{N}-\left(\frac{\sum^N_{j=1}{d_j}}{N}\right)^2}
=\sqrt{\frac{\sum{d^2}}{N}-\left(\frac{\sum{d}}{N}\right)^2}
=\sqrt{\overline{d^2}-\bar{d}^2}\label{eq9}\tag{9}
\end{align}<p>
\begin{align} 
s
=\sqrt{\frac{\sum^N_{j=1}{f_jd_j^2}}{N}-\left(\frac{\sum^N_{j=1}{f_jX_j}}{N}\right)^2}
=\sqrt{\frac{\sum{fd^2}}{N}-\left(\frac{\sum{fd}}{N}\right)^2}
=\sqrt{\overline{d^2}-\bar{d}^2}\label{eq10}\tag{10}
\end{align}<p>
(See Problems 4.15 and 4.17.)<p>
When data is grouped into a frequency distribution whose class intervals have equal size $c$, we have $d_j=cu_j$ or $X_j=A+cu_j$ and result (10) becomes<p>
\begin{align}
s
=c\sqrt{\frac{\sum^K_{j=1}{f_ju_j^2}}{N}-\left(\frac{\sum^K_{j=1}{f_ju_j}}{N}\right)^2}
=c\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
=c\sqrt{\overline{u^2}-\bar{u}^2}\label{eq11}\tag{11}
\end{align}<p>
This last formula provides a very short method for computing the standard deviation and should always be used for grouped data when the class-interval sizes are equal. It is called the _coding method_ and is exactly analogous to that used in Chapter 3 for computing the arithmetic mean of grouped data. (See Problems 4.16 to 4.19.)


---

## Computing the standard deviations from grouped data

__4.12.__ (a) Prove that
\begin{align}
s
= \sqrt{\frac{\sum{X^2}}{N}-\left(\frac{\sum{X}}{N}\right)^2}
= \sqrt{\overline{X^2}-\bar{X}^2}
\end{align}<p>
(b) Use the formula in part (a) to find the standard deviation of the set $12, 6, 7, 3, 15, 10, 18, 5$.

### Solution
(a) By definition,
\begin{align}
s
=\sqrt{\frac{\sum{(X-\bar{X})^2}}{N}}
\end{align}<p>
Then
\begin{align}
s^2
=\frac{\sum{(X-\bar{X})^2}}{N}
=\frac{\sum{(X^2-2\bar{X}X+\bar{X}^2)}}{N}
=\frac{\sum{X^2}=2\bar{X}\sum{X}+N\bar{X}^2}{N}
\end{align}<p>
\begin{align}
=\frac{\sum{X^2}}{N}-2\bar{X}\frac{\sum{X}}{N}+\bar{X}^2=\frac{\sum{X^2}}{N}-2\bar{X}^2+\bar{X}^2=\frac{\sum{X^2}}{N}-\bar{X}^2
\end{align}<p>
\begin{align}
=\overline{X^2}=\bar{X}^2=\frac{\sum{X^2}}{N}-\left(\frac{\sum{X}}{N}\right)2
\end{align}<p>
or
\begin{align}
s
=\sqrt{\frac{\sum{X^2}}{N}-\left(\frac{\sum{X}}{N}\right)^2}
=\sqrt{\overline{X^2}-\bar{X}^2}
\end{align}<p>
Note that in the above summations we have used the abbreviated form, with $X$ replacing $X_j$ and with $\sum$ replacing $\sum^N_{j=1}$.<p>
#### Another method
\begin{align}
s^2
=\overline{(X-\bar{X})^2}
=\overline{X^2-2X\bar{X}+\bar{X}^2}
=\overline{X^2}-\overline{2X\bar{X}}+\overline{\bar{X}^2}
=\overline{X^2}-2X\bar{X}+\bar{X}^2
=\overline{X^2}-\bar{X}^2
\end{align}<p>
(b)
\begin{align}
\overline{X^2}
=\frac{\sum{X^2}}{N}
=\frac{(12)^2+(6)^2+(7)^2+(3)^2+(15)^2+(10)^2+(18)^2+(5)^2}{8}
=\frac{912}{8}=114
\end{align}<p>
\begin{align}
\bar{X}
=\frac{\sum{X}}{N}
=\frac{12+6+7+3+15+10+18+5}{8}=\frac{76}{8}=9.5
\end{align}<p>
Thus
\begin{align}
s
=\sqrt{\overline{X^2}-\bar{X}^2}
=\sqrt{114-90.25}
=\sqrt{23.75}
=4.87
\end{align}<p>
This method should be compared with that of Problem 4.9(a).

In [5]:
p12.b <- c(12, 6, 7, 3, 15, 10, 18, 5)
p9.a.s
(p12.s <- round(sqrt(mean(p12.b^2) - mean(p12.b)^2),2))

[1] 4.87

[1] 4.87

---

__4.13__ Modify the formula of Problem 4.12(a) to allow for frequencies corresponding to the various values of $X$.

### Solution
The appropriate modification is<p>
\begin{align}
s
=\sqrt{\frac{\sum{fX^2}}{N}-\left(\frac{\sum{fX}}{N}\right)^2}
=\sqrt{\overline{X^2}-\bar{X}^2}
\end{align}<p>
As in Problem 4.12(a), this can be established by starting with<p>
\begin{align}
s
=\sqrt{\frac{\sum{f(X-\bar{X})^2}}{N}}
\end{align}<p>
Then
\begin{align}
s^2
=\frac{\sum{f(X-\bar{X})^2}}{N}
=\frac{\sum{f(X^2-2\bar{X}{X}+\bar{X}^2)}}{N}
=\frac{\sum{fX^2}-2\bar{X}\sum{fX}+\bar{X}^2\sum{f}}{N}
\end{align}<p>
\begin{align}
=\frac{\sum{fX^2}}{N}-2\bar{X}\frac{\sum{fX}}{N}+\bar{X}^2
=\frac{\sum{fX^2}}{N}-2\bar{X}^2+\bar{X}^2
=\frac{\sum{fX^2}}{N}-\bar{X}^2
\end{align}<p>
\begin{align}
=\frac{\sum{fX^2}}{N}-\left(\frac{\sum{fX}}{N}\right)^2
\end{align}<p>
or
\begin{align}
s=\sqrt{\frac{\sum{fX^2}}{N}-\left(\frac{\sum{fX}}{N}\right)^2}
\end{align}<p>
Note that in the above summations we have used the abbreviated form, with $X$ and $f$ replacing $X^j$ and $f^j$,$\sum$ replacing $\sum^K_{j=1}$, and $\sum^K_{j=1}f_j=N$.

---

__4.14__ Using the formula of Problem 4.13, find the standard deviation for the data in Table 4.2 of Problem 4.11.

### Solution
The work can be arranged as in the table, where $\bar{X}=\frac{\sum{fX}}{N}=67.45$in, as obtained in Problem 3.15. Note that this method, like that of Problem 4.11, entails much tedious computation. Problem 4.17 shows how the coding method simplifies the calculations immensely.

In [6]:
p14.X <- p11.table[,1]
p14.X.squared <- p14.X^2
p14.f <- p11.table[,4]
p14.f.X.squared <- p14.f*p14.X.squared
p14.table <- matrix( c(p14.X, p14.X.squared, p14.f, p14.f.X.squared), nrow=5)
rownames(p14.table) <- rownames(p11.table)
colnames(p14.table) <- c("Class Mark (X)", "X^2", "Frequency (f)", "fX^2")
p14.table

,Class Mark (X),X^2,Frequency (f),fX^2
60-62,61,3721,5,18605
63-65,64,4096,18,73728
66-68,67,4489,42,188538
69-71,70,4900,27,132300
72-74,73,5329,8,42632


\begin{align}
s
=\sqrt{\frac{\sum{fX^2}}{N}-\left(\frac{\sum{fX}}{N}\right)^2}
=\sqrt{\frac{455\text{,}803}{100}-(67.45)^2}=\sqrt{8.5275}=2.92\text{in}
\end{align}

In [7]:
p14.sum.f.X.squared <- sum(p14.f.X.squared)
p14.N <- sum(p14.f)
p14.X.bar <- sum(p14.f*p14.X)/p14.N
(p14.s <- round(sqrt(p14.sum.f.X.squared/p14.N-p14.X.bar^2),2))

[1] 2.92

---

__4.15__ If $d=X-A$ are the deviations of $X$ from an arbitrary constant $A$, prove that
\begin{align}
s
=\sqrt{\frac{\sum{fd^2}}{N}-\left(\frac{\sum{fd}}{N}\right)^2}
\end{align}

### Solution
Since $d=X-A, X=A,$ and $\bar{X}=A+\bar{d}$(see Problem 3.18), then<p>
\begin{align}
X-\bar{X}=(A+d)-(A+\bar{d})=d-\bar{d}
\end{align}<p>
so that
\begin{align}
s
=\sqrt{\frac{\sum{f(X-\bar{X})^2}}{N}}
=\sqrt{\frac{\sum{f(d-\bar{d})^2}}{N}}
=\sqrt{\frac{\sum{fd^2}}{N}-\left(\frac{\sum{fd}}{N}\right)^2}
\end{align}<p>
using the result of Problem 4.13 and replacing $X$ and $\bar{X}$ with $d$ and $\bar{d}$, respectively.
#### Another method
\begin{align}
s^2
=\overline{(X-\bar{X})^2}=\overline{(d-\bar{d})^2}=\overline{d^2-2\bar{d}d+\bar{d}^2}
\end{align}<p>
\begin{align}
=\overline{d^2}-2\bar{d}^2+\bar{d}^2=\overline{d^2}-\bar{d}^2=\frac{\sum{fd^2}}{N}-\left(\frac{\sum{fd}}{N}\right)^2
\end{align}<p>
and the result follows on taking the positive square root.

---

__4.16__ Show that if each class mark $X$ in a frequency distribution having class intervals of equal size $c$ is coded into a corresponding value $u$ according to the relation $X=A+cu$, where $A$ is a given class mark, then the standard deviation can be written as<p>
\begin{align}
s
=\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
=c\sqrt{\overline{u^2}-\bar{u}^2}
\end{align}

### Solution
This follows at once from Problem 4.15 since $d=X-A=cu$. Thus, since $c$ is a constant,<p>
\begin{align}
s
=\sqrt{\frac{\sum{f(cu)^2}}{N}-\left(\frac{\sum{f(cu)}}{N}\right)^2}
=\sqrt{c^2\frac{\sum{fu^2}}{N}-c^2\left(\frac{\sum{fu}}{N}\right)^2}
=c\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
\end{align}
#### Another method
We can also prove the result directly without using Problem 4.15. Since $X=A+cu, \bar{X}=A+c\bar{u},$ and $X-\bar{X}=c(u-\bar{u})$, then<p>
\begin{align}
s^2
=\overline{(X-\bar{X})^2}
=\overline{c^2(u-\bar{u})^2}
=c^2\overline{(u^2-2\bar{u}u+\bar{u}^2)}
=c^2(\overline{u^2}-2\bar{u}^2+\bar{u}^2)
=c^2(\overline{u^2}-\bar{u}^2)
\end{align}<p>
and
\begin{align}
s
=c\sqrt{\overline{u^2}-\bar{u}^2}
=c\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
\end{align}<p>

---

__4.17__ Find the standard deviation of the hieghts of the students at XYZ University (see Table 2.1) by using (a) the formula derived in Problem 4.15 and (b) the coding method of Problem 4.16.

### Solution
In Tables 4.4 and 4.5, $A$ is arbitrarily chosen as being equal to the class mark $67$. Note that in Table 4.4 the deviations $d=X-A$ are all multiples of the class-interval size $c=3$. This factor is removed in Table 4.5. As a result, the computations in Table 4.5 are greatly simplified (compare them with those of Problems 4.11 and 4.14). For this reason, the coding method should be used wherever possible.

(a) See Table 4.4 below

In [8]:
#load("Saves/Table_2.1.Rdata")
Table_4.4 <- matrix( c(61, 64, 67, 70, 73, -6, -3, 0, 3, 6, 5, 18, 42, 27, 8, -30, -54, 0, 81, 48, 180, 162, 0, 243, 288), nrow=5)
colnames(Table_4.4) <- c("Class Mark (X)", "d=X-A", "Frequency (f)", "fd", "fd^2")
rownames(Table_4.4) <- c("", "", "A", "", "")
Table_4.4
p17.a.N <- sum(Table_4.4[,3])
p17.a.fd.sum <- sum(Table_4.4[,4])
p17.a.fd.squared.sum <- sum(Table_4.4[,5])
(p17.a.s <- round(sqrt(p17.a.fd.squared.sum/p17.a.N-(p17.a.fd.sum/p17.a.N)^2),2))

,Class Mark (X),d=X-A,Frequency (f),fd,fd^2
,61,-6,5,-30,180
,64,-3,18,-54,162
A,67,0,42,0,0
,70,3,27,81,243
,73,6,8,48,288


[1] 2.92

\begin{align}
s
=\sqrt{\frac{\sum{fd^2}}{N}-\left(\frac{\sum{fd}}{N}\right)^2}
=\sqrt{\frac{873}{100}-\left(\frac{45}{100}\right)^2}
=\sqrt{8.5275}=2.92\text{in}
\end{align}

(b) see Table 4.5 below

In [9]:
Table_4.5 <- matrix(c(61,64,67,70,73, -2, -1, 0, 1, 2, 5, 18, 42, 27, 8, -10, -18, 0, 27, 16, 20, 18, 0, 27, 32), nrow=5)
colnames(Table_4.5) <- c("Class Mark (X)", "u=(X-A)/c", "Frequency (f)", "fu", "fu^2")
rownames(Table_4.5) <- c("", "", "A", "", "")
Table_4.5
p17.b.c <- 3
p17.b.N <- sum(Table_4.5[,3])
p17.b.fu.sum <- sum(Table_4.5[,4])
p17.b.fu.squared.sum <- sum(Table_4.5[,5])
(p17.b.s <- round(p17.b.c*sqrt(p17.b.fu.squared.sum/p17.b.N-(p17.b.fu.sum/p17.b.N)^2),2))

,Class Mark (X),u=(X-A)/c,Frequency (f),fu,fu^2
,61,-2,5,-10,20
,64,-1,18,-18,18
A,67,0,42,0,0
,70,1,27,27,27
,73,2,8,16,32


[1] 2.92

\begin{align}
s
=c\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
=3\sqrt{\frac{97}{100}-\left(\frac{15}{100}\right)^2}
=3\sqrt{0.9475}=2.92\text{in}
\end{align}

---

__4.18__ Using coding methods, find (a) the mean and (b) the standard deviation for the wage distribution of the $65$ employees at the P&R Company (see Table 2.5 of Problem 2.3).

### Solution
The work can be arranged simply, as shown in Table 4.6.
(a)
\begin{align}
\bar{X}=A+c\bar{u}=A+c\frac{\sum{fu}}{N}=\$275.00+(\$10.00)\left(\frac{31}{65}\right)=\$279.77
\end{align}
(b)
\begin{align}
s
=c\sqrt{\overline{u^2}-\bar{u}^2}
=c\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
=(\$10.00)\sqrt{\frac{173}{65}-\left(\frac{31}{65}\right)^2}
=(\$10.00)\sqrt{2.4341}=\$15.60
\end{align}

In [10]:
#load("Saves/Table_2.6.Rdata")
Table_4.6 <- matrix(c(255.00, 265.00, 275.00, 285.00, 295.00, 305.00, 315.00, -2, -1, 0, 1, 2, 3, 4, 8, 10, 16, 14, 10, 5, 2, -16, -10, 0, 14, 20, 15, 8, 32, 10, 0, 14, 40, 45, 32), nrow=7)
colnames(Table_4.6) <- c("X", "u", "f", "fu", "fu^2")
rownames(Table_4.6) <- c( "", "", "A", "", "", "", "")
Table_4.6
p18.N <- sum(Table_4.6[,3])
p18.fu.sum <- sum(Table_4.6[,4])
p18.fu.squared.sum <- sum(Table_4.6[,5])
p18.A <- Table_4.6[3,1]
p18.c <- 10
(p18.X.bar <- round(p18.A + p18.c*(p18.fu.sum/p18.N),2))
(p18.s <- round(p18.c*sqrt(p18.fu.squared.sum/p18.N-(p18.fu.sum/p18.N)^2),2))

,X,u,f,fu,fu^2
,255,-2,8,-16,32
,265,-1,10,-10,10
A,275,0,16,0,0
,285,1,14,14,14
,295,2,10,20,40
,305,3,5,15,45
,315,4,2,8,32


[1] 279.77

[1] 15.6

---

__4.19__ Table 4.7 shows the IQ's of 480 school children at a certain elementary school. Using the coding method, find (a) the mean and (b) the standard deviation.

In [11]:
p19.X <- c( 70, 74, 78, 82, 86, 90, 94, 98, 102, 106, 110, 114, 118, 122, 126)
p19.f <- c(4, 9, 16, 28, 45, 66, 85, 72, 54, 38, 27, 18, 11, 5, 2)
Table_4.7 <- t(matrix(c(p19.X, p19.f),ncol=2))
rownames(Table_4.7) <- c("Class mark (X)", "Frequency (f)")
Table_4.7

Class mark (X),70,74,78,82,86,90,94,98,102,106,110,114,118,122,126
Frequency (f),4,9,16,28,45,66,85,72,54,38,27,18,11,5,2


### Solution
The intelligence quotient is

\begin{align}
\text{IQ}=\frac{\text{mental age}}{\text{chronological age}}
\end{align}

expressed as a percentage. For example, an 8-year-old child who (according to certain educational procedures) has a mentality equivalent to that of a 10-year-old child would have an IQ of $10/8=1.25=125\%$, or simply $125$, the $\%$ sign being understood.

To find the mean and standard deviation of the IQ's in Table 4.7, we can arrange the work as in Table 4.8

(a)
\begin{align}
\bar{X}=A+c\bar{u}=A+c\frac{\sum{fu}}{N}=94+4\left(\frac{236}{480}\right)=95.97
\end{align}
(b)
\begin{align}
s
=c\sqrt{\overline{u^2}-u^2}
=c\sqrt{\frac{\sum{fu^2}}{N}-\left(\frac{\sum{fu}}{N}\right)^2}
=4\sqrt{\frac{3404}{480}-\left(\frac{236}{480}\right)^2}
=4\sqrt{6.8499}=10.47
\end{align}

In [12]:
p19.c <- 4
p19.A <- as.vector(Table_4.7[1,7])
p19.N <- sum(Table_4.7[2,])
p19.u <- c( -6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8)
p19.fu <- p19.f*p19.u
p19.fu.sum <- sum(p19.fu)
(p19.X.bar <- round(p19.A + p19.c*(p19.fu.sum/p19.N),2))
p19.fu.squared.sum <- sum(p19.f*p19.u^2)
(p19.s <- round(p19.c*sqrt(p19.fu.squared.sum/p19.N-(p19.fu.sum/p19.N)^2),2))

[1] 95.97

[1] 10.47